src/components/Select/index.tsx

```diff
@@ -30,11 +30,10 @@ const SelectLabel = ({
 }: SelectLabelProps): JSX.Element => (
   <Listbox.Label
     className={clsx(
-      'block',
-      'text-sm',
-      'font-medium',
-      'text-textSecondary',
-      'mb-1',
+      // TODO: placeholder color
```

Comment:
@tomjeatt 
I'd say we can remove this comment since it's only relevant to the staking UI.